Update README.md (#23033)
kubernetes/website: README.md

## Running the website locally using Hugo

> Windows 10, make is not available by default. To run hugo without container support,
> Make sure to install the Hugo extended version specified by the `HUGO_VERSION` environment variable in the [`netlify.toml`](netlify.toml#L10) file.

Reviewer: ```suggestion
You can run Hugo directly (not inside a container). You need to install the version ofHugo that is specified by the `HUGO_VERSION` environment variable in the [`netlify.toml`](netlify.toml#L10) file.
```

To build and test the site locally, run:

```bash

> hugo server --buildFuture
> #Listens on port 1313, 

Reviewer: ```suggestion
make serve

# If you are using a platform such as Windows that does not provide the
# "make" utility, manually verify that you have the dependencies in place
# and then run:
hugo server --buildFuture
```

## Running the website locally using Hugo

> You can run Hugo directly (not inside a container). You need to install the version ofHugo that is specified by the `HUGO_VERSION` environment variable in the [`netlify.toml`](netlify.toml#L10) file.

Reviewer 2: I think the original text is clear (install the extended version).
Is it confusing that the sentence points the reader to the `netlify.toml` file?

> You can run Hugo directly (not inside a container). You need to install the version ofHugo that is specified by the `HUGO_VERSION` environment variable in the [`netlify.toml`](netlify.toml#L10) file.

Reviewer: I saw a typo (sorry)
```suggestion
You can run Hugo directly (not inside a container). You need to install the version of Hugo that is specified by the `HUGO_VERSION` environment variable in the [`netlify.toml`](netlify.toml#L10) file.
```

> You can run Hugo directly (not inside a container). You need to install the version of Hugo that is specified by the `HUGO_VERSION` environment variable in the [`netlify.toml`](netlify.toml#L10) file.

Reviewer 2: HI @forvaidya .
I think this change needs to mention that the extended version of Hugo is required.
Hugo publishes more than one type of a given version.
From the theme's documentation:
```
Install a recent release of the Hugo "extended" version (we recommend version 0.53 or later).
```

To build and test the site locally, run:

```bash
make serve

> 

Reviewer 2: What do you think about listing the different commands in separate code blocks or list only
`hugo server --buildFuture`? The `make serve` target performs an additional check.

Reviewer: That extra check from `make serve` is useful and if available I'd steer readers to use it.
